pkg/pdapi/fake_pdapi.go

```diff
@@ -270,3 +271,13 @@ func (c *FakePDClient) GetAutoscalingPlans(strategy Strategy) ([]Plan, error) {
 	}
 	return nil, nil
 }
+
+func (c *FakePDClient) GetRecoveringMark() (bool, error) {
+	action := &Action{}
+	result, err := c.fakeAPI(GetRecoveringMarkActionType, action)
+	if err != nil {
+		return result.(bool), err
```

Comment:
I suggest to return `false, err`, I am not sure whether the `result` is `nil` if `err != nil`.

Reply:
fix in https://github.com/pingcap/tidb-operator/pull/4812/commits/30ac10ba55a7b72b5780652c5163c320d99a4453